packages/shared/src/components/squads/SquadEmptyScreen.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react';
+import { cloudinary } from '../../lib/image';
+import { Image } from '../image/Image';
+import { FlexCentered } from '../utilities';
+
+function SquadEmptyScreen(): ReactElement {
+  return (
+    <FlexCentered className="flex-col tablet:self-center py-14 w-full laptop:h-full text-center laptop:min-h-[calc(100vh-442px)]">
```

Comment:
We do have a `EmptyScreenContainer` did that one not work?

Reply:
It has too many predefined styles and would need to override with `!important`. Also it adds `100vh` which then adds a massive scroll to the page due to squad page header. Also did not use `EmptyScreenTitle` because it also differs from this one.

Reply:
Would be happy to use those if design standardised it but I feel whole squad direction has a little different design goals from the rest of the pages. 

If you want I can check with design on all of this?

Reply:
Nah it's ok for now. 
Missed the part where it spans so we need the full height minus header :)

If this becomes final we can revamp the tech debt 🙈